site/content/integrate/getting-started/how-should-i-integrate.md

```diff
@@ -28,7 +28,7 @@ To build a richer bot integration, you can make full use of the [Mattermost REST
 
 ### I want to build an interactive app
 
-That's no problem. Our Apps provide a framework with utilities like a KV store, bot account creation, OAuth permission scopes, and rich UX entry points.
+That's no problem. Our Apps provide a framework with utilities like a key-value store, bot account creation, OAuth permission scopes, and rich UX entry points.
```

Comment:
Honestly the OAuth permission scopes are pretty limited at the moment, so I don't know if we want to advertise them. Thoughts on this @hanzei?

Reply:
They are one selling point which we will improve in the future. 1/5 to keep them in.

Reply:
It depends on the speed at which we're iterating tbh. Might be helpful to leave it in?